### DISCUSSION TITLE
v3.1 release timeline?

### DISCUSSION THREAD

**Discussion by qswinson** [dexie/Dexie.js#1313] [General]
This project looks amazing! I'm very excited about the potential of `useLiveQuery`. However, I'm hesitating to start integrating with this feature due to its alpha status and the fact that it has been in alpha for six months now.

Are there known issues that would make this code not production ready? I did search the github issues and only found two possible issues, neither of which I think would be a show stopper for me.

Is there a release timeline for v3.1?

Thanks!

**Comment by dfahlander** [comment]
Hi! The latest alpha is used by lots of people. We've found bugs and fixed them. I'm not worried about stability anymore in this version. The way to import and use 'dexie-react-hooks' is also stable. There might be a change in how to import { liveQuery } from 'dexie' as I would like to extract it from dexie module into a separate module.

Then there is also a time aspect in releasing a new version.

Conclusion: You can consider dexie@3.1.0-alpha.10 stable. I will change status to beta soon due to the proposed refactoring of `liveQuery()` into its own module - not due to stability issues.

**Comment by dfahlander** [comment]
Just a note: Dexie 3.2.0 is now in release candidate since last week. No issues reported far so good, so I'm hoping it can be released officially very soon. That would be a real milestone as it will be the first official release with liveQuery support.
